db/schema.rb

```diff
@@ -512,6 +513,7 @@
     t.index ["access_level"], name: "index_users_on_access_level"
     t.index ["deleted_at"], name: "index_users_on_deleted_at"
     t.index ["organization_id"], name: "index_users_on_organization_id"
+    t.index ["teleconsultation_phone_number"], name: "index_users_on_teleconsultation_phone_number"
```

Comment:
@prabhanshuguptagit looks like this index was missed from the schema when the migration was added. Perhaps the migration was run on a local db and then modified to add an index? In any case, I want to confirm that this index _should_ be here.

Reply:
I think I added the index later and missed committing the schema, yes. This is correct.